seal_rookery/search.py

```diff
@@ -36,7 +36,7 @@ def seal(court: str, size: SIZES = ImageSizes.MEDIUM) -> Optional[str]:
     :param size: The size of the seal you want a URL for..
     :return A URL to the image.
     """
-    if court not in seals.keys():
+    if court not in seals:
```

Comment:
Fix:

```
seal_rookery/search.py:39:8: SIM118 Use `key not in dict` instead of `key not in dict.keys()`
   |
37 |     :return A URL to the image.
38 |     """
39 |     if court not in seals.keys():
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^ SIM118
40 |         return None
41 |     if size == ImageSizes.ORIGINAL:
   |
   = help: Remove `.keys()`
```

https://docs.astral.sh/ruff/rules/in-dict-keys/